packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -750,7 +710,7 @@ export enum UsageFlags {
 interface SdkExampleBase {
   kind: string;
   name: string;
-  description: string;
+  doc: string
```

Comment:
I made this breaking change so it's `.doc` everywhere, @tadelesh are you ok with that?

Reply:
it's ok to change this.